salesagility/SuiteCRM
# fix bug on clicking Update when changing Date parameter field in report detail view

Fixes display bug in report's DetailView screen with date as parameter

## Description
Steps to reproduce the bug:
1. Create a report with date field as parameter
2. Go to the Report's DetailView screen and click on "Update"
Before clicking "Update":
![image](https://user-images.githubusercontent.com/34060805/34168462-3f0e47e6-e4ed-11e7-9d96-d0a55a865422.png)

After clicking "Update":
![image](https://user-images.githubusercontent.com/34060805/34168474-43877f04-e4ed-11e7-96de-eec36df0cee3.png)

## Motivation and Context
Fixes the bug described above

## How To Test This
1. Create a report with date field as parameter
2. Go to the Report's DetailView screen and click on "Update"
3. Date fields should remain the same as before clicking "Update"

## Types of changes

### Final checklist
